python/ml4ir/applications/classification/pipeline.py

```diff
@@ -69,7 +69,7 @@ def get_relevance_model(self, feature_layer_keys_to_fns={}) -> RelevanceModel:
             learning_rate=self.args.learning_rate,
             learning_rate_decay=self.args.learning_rate_decay,
             learning_rate_decay_steps=self.args.learning_rate_decay_steps,
-            gradient_clip_value=self.args.gradient_clip_value,
+            clipvalue=self.args.gradient_clip_value,
```

Comment:
@balikasg Why are you not passing `with_exp_decay` here?

Also, looks like there is mismatch between `learning_rate` and `initial_learning_rate`. Maybe I am missing something actually.